src/commands/settings.py

```diff
@@ -19,3 +33,23 @@ def handle_hide_cuisine(update, context):
     cuisine_to_hide = parse_callback(update.callback_query.data)[1]
     updated_hidden_cuisine = update_hidden_cuisine(update.effective_chat.id, cuisine_to_hide)
     update.callback_query.edit_message_reply_markup(reply_markup=hidden_cuisine_kb(updated_hidden_cuisine))
+
+
+def handle_notification(update, context):
+    chat_id = update.effective_chat.id
+    bf_sub = get_subscribe_setting(BREAKFAST, chat_id)
```

Comment:
can combine these two queries to one using destructuring

Reply:
Hmm not sure if there is a way to query two columns at the same time?